src/hostCommands/testCompile.ts

```diff
@@ -36,10 +36,15 @@ export async function testCompileHandler(context: utils.ExtensionCommandContext)
 }
 
 export async function testCompile(fsPath: string) {
-	// TODO check if testCompile-able?
-	if (!fs.statSync(fsPath).isFile()) return;
+	if (!fs.statSync(fsPath).isFile()) {
```

Comment:
Avoid using the synchronous IO operations. Better:
```suggestion
	const isFile = await fs.stat(fsPath);
	if (!isFile) {
```

Reply:
Using `await` where possibly makes sense - I'll apply this change.

Reply:
Using `await` where possibility makes sense. I've updated my code to include that.

Reply:
Thanks!